backend/routes/assignments.js

```diff
@@ -103,19 +104,47 @@ router.delete('/:id', auth, async (req, res) => {
     const userIdStr = userId.toString()
     const user = await userData.getUser(userIdStr)
 
-    if(user.role !== "teacher") {
+    if (user.role !== "teacher") {
         res.sendStatus(403)
         return
     }
 
     try {
-        await courseData.removeAssignment( courseId, id )
-        res.status(200).json({deleted: true})
+        await courseData.removeAssignment(courseId, id)
+        res.status(200).json({ deleted: true })
         return
-    } catch(e) {
+    } catch (e) {
         console.log(e)
         res.sendStatus(400)
     }
 })
 
+/**
+ * Fetch grade for a student with assignmentId
+ * Route: GET
+ * Params: assignmentId
+ * Query: id
+ */
+router.get('/grades/:assignmentId', auth, async (req, res) => {
```

Comment:
Did we want the query to be optional? If so we can't use `error.str(studentId)` as this will enforce that the user must provide a query. Just wasn't sure

Reply:
student id is a mandatory field, we need id to fetch assignments. I was just following the route path specified in db docs. Let me know if we should change the parameter representation